tgui/packages/tgui/interfaces/Mecha/ModulesPane.tsx

```diff
@@ -476,17 +494,49 @@ const SnowflakeSyringe = (props) => {
           }
         />
       </LabeledList.Item>
-      <LabeledList.Item label={'Reagent control'}>
+      <LabeledList.Item label={'Syntheizing'}>
```

Comment:
```suggestion
      <LabeledList.Item label="Synthesizing">
```
sorry i did not see this before